security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,153 +12,28 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
-[id="cert-manager-operator-release-notes-1.12.1"]
-== Release notes for {cert-manager-operator} 1.12.1
+[id="cert-manager-operator-release-notes-1.13"]
+== Release notes for {cert-manager-operator} 1.13
 
-Issued: 2023-11-15
+Issued: 2024-xx-xx
 
-The following advisory is available for the {cert-manager-operator} 1.12.1:
+The following advisory is available for the {cert-manager-operator} 1.13:
 
 * link:https://access.redhat.com/errata/RHSA-2023:6269-02[RHSA-2023:6269-02]
 
-Version `1.12.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.12.5`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.12/#v1125[cert-manager project release notes for v1.12.5].
+Version `1.13` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.4`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.12/#v1125[cert-manager project release notes for v1.13].
```

Comment:
Upstream version will be `v1.13.3` instead of `v1.13.4`.

Reply:
Yes, and need to change the link to `https://cert-manager.io/docs/releases/release-notes/release-notes-1.13#v1133`